modules/monitoring-adding-a-secret-to-an-alertmanager-configuration.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-adding-a-secret-to-an-alertmanager-configuration_{context}"]
+= Adding a secret to an Alertmanager configuration
+
+You can add secrets to the Alertmanager configuration for core platform monitoring components by editing the `cluster-monitoring-config` config map for the `openshift-monitoring` project.
+You can add secrets to the Alertmanager configuration for user-defined projects by editing the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` project.
+
+[NOTE]
+====
+To configure Alertmanager for user-defined projects to reference the same secret as exists in the `openshift-monitoring` project, you must create the same secret in both the `openshift-monitoring` project and the `openshift-user-workload-monitoring` project.
+==== 
+
+After you add a secret to the config map, the secret is mounted as a volume at `/etc/alertmanager/secrets/<secret_name>` within the `alertmanager` container for the Alertmanager pods.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have created the secret for the endpoint receiver.
```

Comment:
```suggestion
* You have created the secret for configuring in Alertmanager.
```

Reply:
Fixed and added separate prerequisite items for core and UWM sections.